lms/services/grading_info.py

```diff
@@ -20,11 +22,17 @@ def __init__(self, _context, request):
         self._db = request.db
         self._authority = request.registry.settings["h_authority"]
 
-    def get_by_assignment(self, application_instance, context_id, resource_link_id):
+    def get_students_for_grading(
```

Comment:
We don't use `GradingInfos` outside the context of the "grading bar".

Moved setting all of that up here.

I reckon this could be improved by making this is the "ToolbarGrading" service and moving a few more bits here. I'll leave that for later.